packages/excalidraw/tests/__snapshots__/history.test.tsx.snap

```diff
@@ -12268,10 +12268,7 @@ exports[`history > singleplayer undo/redo > should create new history entry on e
   "openMenu": null,
   "openPopup": null,
   "openSidebar": null,
-  "originSnapOffset": {
-    "x": 0,
-    "y": 0,
-  },
+  "originSnapOffset": null,
```

Comment:
Because, now, we are always running a `setActiveTool` after each paste. Previously, some cases early-exited.